samples/javascript_nodejs/13.basic-bot/dialogs/luisHelper.js

```diff
@@ -17,37 +16,38 @@ class LuisHelper {
             const recognizer = new LuisRecognizer({
                 applicationId: process.env.LuisAppId,
                 endpointKey: process.env.LuisAPIKey,
-                endpoint: `https://${process.env.LuisAPIHostName}`
+                endpoint: `https://${ process.env.LuisAPIHostName }`
             }, {}, true);
 
             const recognizerResult = await recognizer.recognize(context);
 
             const intent = LuisRecognizer.topIntent(recognizerResult);
-            const score = recognizerResult.luisResult.topScoringIntent.score;
-            
+
+            bookingDetails.intent = intent;
```

Comment:
What's this for? I don't think we use the intent in this sample...

Reply:
Not sure. This code is based on the changes that were done to Ben's update. I'm only removing samples and adding folder + readmes for additional samples we are migrating from another repo in this PR. You might want to check with Ben/Scott.